src/library/scala/concurrent/duration/Duration.scala

```diff
@@ -173,17 +173,17 @@ object Duration {
    *
    * The particular comparison semantics mirror those of Double.NaN.
    *
-   * '''''Use `eq` when checking an input of a method against this value.'''''
+   * '''''Use [[Duration.eq]] when checking an input of a method against this value.'''''
    */
   val Undefined: Infinite = new Infinite {
     override def toString = "Duration.Undefined"
-    override def equals(other: Any) = false
+    override def equals(other: Any):Boolean = false
```

Comment:
because then we have to discuss the irregular lack of a space.

Reply:
> because then we have to discuss the irregular lack of a space.
I think we should make use of scalafmt @olafurpg

Reply:
That comes up every so often, but they claim this code base is too free-wheelin'.